README.md

```diff
@@ -782,12 +811,12 @@ Examples exist on [codesandbox.io][examples]:
 
 If you would like to add an example, follow these steps:
 
-1. Fork [this codesandbox](http://kcd.im/ds-example)
-2. Make sure your version (under dependencies) is the latest available version.
-3. Update the title and description
-4. Update the code for your example (add some form of documentation to explain
-   what it is)
-5. Add the tag: `downshift:example`
+1.  Fork [this codesandbox](http://kcd.im/ds-example)
+2.  Make sure your version (under dependencies) is the latest available version.
+3.  Update the title and description
+4.  Update the code for your example (add some form of documentation to explain
+    what it is)
+5.  Add the tag: `downshift:example`
```

Comment:
I'm not sure I understand what's causing some of these changes.

Everything other than the chunk added about the default event handlers is being added automatically after I run and complete the `add-contributor` script. I can split the changes into two commits to show this if need be.

Some of these spacing changes are innocuous, but it seems like the automated changes being made to the contributors table are swapping out absolute paths with local paths (e.g. `https://github.com/paypal/downshift/commits?author=kentcdodds "Code"` is replaced with `/paypal/downshift/commits?author=kentcdodds "Code"`, and so on).

If someone has a clue as to why this is happening, I'd like to remove these undesired changes from this pull request.

Reply:
You didn't add the spaces manually? That's interesting...

Also, I'm not sure what caused the paths to change from absolute to local, but I have seen this behavior in previous add-contributor commits.

Reply:
Yeah, I'm pretty sure that prettier was updated since it was last run on this file or something. I'm not really concerned, but I am a bit confused by that.

I'm pretty sure what caused this [this pr](https://github.com/jfmengels/all-contributors-cli/pull/84). I've released a fix to all-contributors-cli. If you reinstall dependencies then run `./node_modules/.bin/kcd-scripts contributors generate` it should get fixed up.

Reply:
Nice! Thank you, @austintackaberry and @kentcdodds. The change made to the executable module worked. Should be good for another review now :)